code/game/objects/effects/decals/cleanable/misc.dm

```diff
@@ -89,10 +89,12 @@
 	addtimer(CALLBACK(src, .proc/Decay), 24 SECONDS)
 
 /obj/effect/decal/cleanable/greenglow/proc/Decay()
+	var/datum/component/radioactive/R = src.GetComponent(/datum/component/radioactive)
```

Comment:
```suggestion
	var/datum/component/radioactive/R = GetComponent(/datum/component/radioactive)
```

Reply:
note that src. was required for this to work here. Tried it without and it didn't want to work